### PR TITLE
added social authentication

### PR DESCRIPTION
**Pull request by nerko937**

### CODE REVIEW

File: example_project/settings.py [hunk 1]
```diff
@@ -138,6 +146,33 @@
...
+SOCIALACCOUNT_PROVIDERS = {
+    'google': {
+        'SCOPE': [
+            'profile',
+            'email',
+        ],
+        'AUTH_PARAMS': {
+            'access_type': 'online',
+        }
+    }
```
Comment (robertmusil) [inline example_project/settings.py:173]: missing github here, no?

Comment (nerko937) [reply example_project/settings.py:173]: I've skipped it intentionally, quote from docs:

> If you want more than just read-only access to public data, specify the scope as follows. See https://developer.github.com/v3/oauth/#scopes for details.
```If you want more than just read-only access to public data, specify the scope as follows. See https://developer.github.com/v3/oauth/#scopes for details.

SOCIALACCOUNT_PROVIDERS = {
    'github': {
        'SCOPE': [
            'user',
            'repo',
            'read:org',
        ],
    }
}
```

I think read-only access is enough.

File: users/models.py [hunk 2]
```diff
@@ -25,7 +25,6 @@ def create_superuser(self, email, password=None):
         Creates and saves a superuser with the given email and password.
         """
         user = self.create_user(email, password)
-        user.is_active = True
```
Comment (robertmusil) [inline users/models.py]: this should stay.

File: users/urls.py [hunk 3]
```diff
@@ -1,9 +1,30 @@
...
+    # google
+    path('google/login/', GoogleLogin.as_view(), name='google_login'),
+    path('google/callback/', google_callback, name='google_callback'),
+    path('google/url/', google_oauth2_login, name='google_url'),
+    # github
+    path('github/login/', GithubLogin.as_view(), name='github_login'),
+    path('github/callback/', github_callback, name='github_callback'),
+    path('github/url/', github_oauth2_login, name='github_url'),
+    # misc
+    path('account-inavtive/', account_inactive, name='account_inactive'),
```
Comment (robertmusil) [inline users/urls.py]: is it still used?

**Review by robertmusil — APPROVED** [review]

**Pull request merged**